### PR TITLE
Thur Evening / Wed Morning Fixes

### PR DESCRIPTION
**Pull request by tjstalcup**

### CODE REVIEW

File: src/sass/page-template-styles.scss [hunk 1]
```diff
@@ -397,6 +433,7 @@
   .card,
   .check,
   .paypal {
+    flex-basis: 30%;
```
Comment (bryancasler) [inline src/sass/page-template-styles.scss:436]: Try calc(100% / 3) for perfectly 1/3 buttons.

File: src/sass/page-template-variables.scss [hunk 2]
```diff
@@ -297,7 +297,7 @@
     // CLICK TO EXPAND
     --theme-click-to-expand-copy-height-before-fade-starts: 100px;
     --theme-click-to-expand-copy-total-height: 450px;
-    --theme-click-to-expand-cta-image-url: url("https://aaf1a18515da0e792f78-c27fdabe952dfc357fe25ebf5c8897ee.ssl.cf5.rackcdn.com/1839/btn+more.png?v=1614305726000");
+    --theme-click-to-expand-cta-image-url: url("https://aaf1a18515da0e792f78-c27fdabe952dfc357fe25ebf5c8897ee.ssl.cf5.rackcdn.com/1839/more-btn.png?v=1614773932000");
```
Comment (bryancasler) [inline src/sass/page-template-variables.scss:300]: Did you delete the old (low res) image asset off their account?

**Review by bryancasler — CHANGES_REQUESTED** [review]

File: src/sass/page-template-styles.scss (hunk 1 above)
Comment (tjstalcup) [reply src/sass/page-template-styles.scss]: Done

File: src/sass/page-template-variables.scss (hunk 2 above)
Comment (tjstalcup) [reply src/sass/page-template-variables.scss:300]: Yes

**Pull request merged**